config/initializers/rack_profiler.rb

```diff
@@ -0,0 +1,6 @@
+if Rails.env == 'development'
```

Comment:
also: profiling did noticeably slow indexing throughput, FWIW (both RubyProf and New Relic).  i don't have the numbers handy, i think there are actually some recorded in some of the github issues, but i think we saw up to a 5-10x slowdown on indexing workers when profiling was enabled.  we typically had to run it for limited windows to collect enough info to debug the memory issues we were running into.  if we left profiling on full time, indexing would've taken something like a month instead of 3 days.

i have neither anecdotal nor scientific evidence for noticeable regular application slow down when new relic was enabled (which is the only profiling we've left on in the app for any extended period).  argo's slow enough anyway that i didn't really notice slowdown from new relic.  and we weren't collecting performance data before new relic, so i have nothing to judge its numbers against.